### ISSUE TITLE
flutter create -a kotlin <project> still generate java code

### ISSUE THREAD

**Issue by leagmain** [flutter/flutter#23212]
### flutter version
```
Flutter 0.10.0 • channel dev • https://github.com/flutter/flutter.git
Framework • revision d954ae6850 (8 days ago) • 2018-10-09 16:21:06 -0400
Engine • revision bf377896ff
Tools • Dart 2.1.0-dev.6.0.flutter-a50dfd6698
```

### issue command
```
flutter create -i swift -a kotlin projectA
```
the `projectA` contains Java version of Android code and ObjC version of iOS code.

## Logs
![image](https://user-images.githubusercontent.com/4177599/47090746-0a804900-d256-11e8-932e-729fe4c704a6.png)
![image](https://user-images.githubusercontent.com/4177599/47090901-664ad200-d256-11e8-9045-41dabf74e5e6.png)

## flutter doctor -v
```
[√] Flutter (Channel dev, v0.10.0, on Microsoft Windows [Version 10.0.17134.345], locale en-US)
    • Flutter version 0.10.0 at C:\Users\leagm\Applications\flutter
    • Framework revision d954ae6850 (8 days ago), 2018-10-09 16:21:06 -0400
    • Engine revision bf377896ff
    • Dart version 2.1.0-dev.6.0.flutter-a50dfd6698

[!] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at C:\Users\leagm\AppData\Local\Android\sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)
    X Android license status unknown.

[√] Android Studio (version 3.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 29.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1136-b06)

[√] VS Code (version 1.28.1)
    • VS Code at C:\Users\leagm\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 2.19.0

[√] VS Code, 64-bit edition (version 1.28.1)
    • VS Code at C:\Program Files\Microsoft VS Code
    • Flutter extension version 2.19.0

[√] Connected device (1 available)
    • MIX 2 • 192.168.0.101:5555 • android-arm64 • Android 8.0.0 (API 26)

! Doctor found issues in 1 category.
```

**Comment by maks** [comment]
I think this has been fixed and can be closed, using current stable Flutter v1.2.1 running `flutter create -a kotlin projk` produces:
` projk/android/app/src/main/kotlin/com/example/projk/MainActivity.kt (created)`
as expected.

**Comment by jmagman** [comment]
The GeneratedPluginRegistrant files are still in java and Objective-C but that's by design.

Are you still seeing this in a recent Flutter build?
